winforms/src/toga_winforms/app.py

```diff
@@ -19,6 +17,12 @@
 from .libs.proactor import WinformsProactorEventLoop
 from .window import Window
 
+# A reference to WindowsBase is needed for Dispatcher
+clr.AddReference(
+    "c:\\Program Files\\Reference Assemblies\\Microsoft\\Framework\\v3.0\\WindowsBase.dll"
+)
```

Comment:
Not sure where you've got this assembly from, but it doesn't exist on my Win10 machine.